examples/theater-metadata-instance.ttl

```diff
@@ -0,0 +1,319 @@
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Lisa Geller" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Video Director"@en, "Video-Regisseur"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Pius Neumaier" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Lighting Designer"@en, "Lichtdesigner"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Moritz Fettinger" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Dramaturg"@en, "Dramaturgin"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Sarah Mössner" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Band Member"@en, "Bandmitglied"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Jonas Horche" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Band Member"@en, "Bandmitglied"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Tilman Herpichböhm" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Band Member"@en, "Bandmitglied"@de ;
+                                 drk:performer   [ a           schema:Person ;
+                                                   schema:name "Niklas Rehle" ; ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Costume Provider"@en ;
+                                 drk:performer   [ a             schema:Organization ;
+                                                   schema:name   "Replace_with_Organizations_name" ;
+                                                   schema:sameAs <Replace_with_Organizations_Website_URL> ] ] ;
+    drk:creatorRole            [ a               drk:ProductionRole ;
+                                 schema:roleName "Sponsor"@en, "Funder"@en ;
+                                 drk:performer   [ a             schema:Organization ;
+                                                   schema:name   "Replace_with_Organizations_name" ;
```

Comment:
Here and in some other places below, is it intended to have such placeholder-like values here?

Reply:
Replaced!